### ISSUE TITLE
API not reflecting revoked access

### ISSUE THREAD

**Issue by MaxHasADHD** [trakt/api-help#112]
I wanted to test what error I'd receive if I revoked access to an app I'm already signed into. I did this on 2 accounts and both times I'm getting success codes for removing history and adding history. These changes aren't reflected on Trakt though so it is being blocked from syncing, but at least from the time I revoked the tokens to now the API isn't letting the app know it has been unauthenticated and need to prompt for log in again.

**Comment by rectifyer** [comment]
I'm not able to reproduce this issue. Please post the sample code you're using to revoke the token. The API will always return a `200` success, even with invalid token info, for security reasons.

**Comment by MaxHasADHD** [comment]
@rectifyer I'm not using the API to revoke the token. I signed into my mobile app, and then on Trakt's website I revoked my app. Then back in my app I fetch history and ratings, or make API calls to add new history, etc with the already-cached token. The API says the actions that should require authentication are succeeding I believe, even though the app no longer has permission to make those calls. If I look on Trakt though the data doesn't change. So this could lead to the app and site being out of sync but the app not know it is supposed to ask for authentication again.
